server/plugin/command.go

```diff
@@ -336,7 +326,8 @@ func (p *Plugin) handleSubscribesAdd(_ *plugin.Context, args *model.CommandArgs,
 				if notificationOffRepoOwner != owner {
 					return fmt.Sprintf("--exclude repository  %s is not of subscribed organization .", NotificationOffRepo)
 				}
-				if err := p.StoreExcludedNotificationRepo(val); err != nil {
+				flags.ExcludeRepos = append(flags.ExcludeRepos, val)
+				if err := p.SubscribeOrg(ctx, githubClient, args.UserId, owner, args.ChannelId, features, flags); err != nil {
```

Comment:
Is this statement correctly placed here? What about the case where no excluded repos are defined?

Reply:
In that case, it won't be part of the flow.

Reply:
Right, this loop won't be entered if `excludeRepo` is empty